testing/cucumber/features/linters/unique_scenario_names.feature

```diff
@@ -0,0 +1,124 @@
+Feature: Unique scenario names
+
+  As a reader of documentation
+  I want each scenario to have a unique name within the same feature file, even if they were created using a Scenario Outline or Rule.
+  So that each scenario describes a specific aspect of the application's functionality.
+
+  Scenario: Linting (No Duplicates within Feature including Rules)
```

Comment:
may be `Linting (Good: No Duplicates within Feature including Rules)`

Reply:
Ok! Modified